src/main/java/com/jensjansson/ce/views/main/MainView.java

```diff
@@ -63,74 +62,63 @@ private Component createSideMenu() {
         H2 header = new H2("Collaboration Engine Demo");
         header.addClassNames("flex", "items-center", "h-xl", "m-0", "px-m",
                 "text-m");
+
         menu = createMenuLinks();
-        HorizontalLayout avatar = createMenuAvatar();
+        Footer footer = createMenuFooter();
 
-        Div layout = new Div(header, menu, avatar);
-        layout.addClassNames("bg-contrast-5", "flex", "flex-col",
-                "items-strech", "min-h-full");
-        return layout;
+        com.vaadin.flow.component.html.Section section = new com.vaadin.flow.component.html.Section(header, menu, footer);
+        section.addClassNames("bg-base", "flex", "flex-col", "items-stretch", "min-h-full");
+        return section;
     }
 
-    private Div createMenuLinks() {
-        Div menu = new Div();
+    private Nav createMenuLinks() {
+        Nav menu = new Nav();
+        menu.addClassNames("mb-l");
+
+        menu.add(
+                createLink(VaadinIcon.EDIT, "Employees", EmployeesView.class),
+                createLink(VaadinIcon.PICTURE, "About", AboutView.class)
+        );
 
-        menu.add(createLink(VaadinIcon.EDIT, "Employees", EmployeesView.class),
-                createLink(VaadinIcon.PICTURE, "About", AboutView.class));
-        menu.addClassNames("p-s", "box-border", "flex", "flex-col",
-                "flex-grow");
-        menu.setWidthFull();
         return menu;
     }
 
     private RouterLink createLink(VaadinIcon vaadinIcon, String text,
             Class<? extends Component> view) {
         RouterLink link = new RouterLink(null, view);
+
         Icon icon = vaadinIcon.create();
-        icon.addClassNames("box-border", "icon-m", "me-s", "p-xs");
+        icon.addClassNames("box-border", "icon-s", "me-s");
+
         Span span = new Span(text);
         span.addClassNames("text-s", "font-medium");
+
         link.add(icon, span);
-        link.addClassNames("rounded-s", "h-m", "px-s", "mx-s", "flex",
-                "items-center", "box-border");
+        link.addClassNames("flex", "h-m", "items-center", "mx-s", "px-s", "relative", "text-secondary");
         return link;
     }
 
-    private HorizontalLayout createMenuAvatar() {
+    private Footer createMenuFooter() {
         avatar = new Avatar(localUser.getName(), localUser.getImage());
-        avatar.addClassNames("pointer");
+        avatar.addClassNames("mr-xs");
+        avatar.addThemeVariants(AvatarVariant.LUMO_XSMALL);
+
         userLabel = new Span(localUser.getName());
-        HorizontalLayout layout = new HorizontalLayout(avatar, userLabel);
-        layout.addClassNames("p-m", "pointer");
-        layout.setAlignItems(FlexComponent.Alignment.CENTER);
-        layout.addClickListener(event -> {
-            layout.getUI().ifPresent(ui -> ui.navigate(YourProfileView.class));
-        });
```

Comment:
This removes the possibility to go back to the profile selection view. Was it on accident that it was removed, or some reason behind it? I would add this back, and the pointer class to layout and avatar.

Reply:
Added clicking on name/avatar back.